.github/workflows/test.yml

```diff
@@ -1,32 +1,12 @@
-# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
-name: Lint, build and test
+name: ci
 on:
-  pull_request:
+  push:
+    branches-ignore:
+      - main
+    paths-ignore:
+      - '**.md'
+
 jobs:
-  build:
-    name: Lint, build and test
-    runs-on: ubuntu-latest
-    timeout-minutes: 3
```

Comment:
I'm curious why we had this, can it be the reason for the stucked workflow?
![image](https://github.com/fingerprintjs/fingerprintjs-pro-svelte/assets/1009415/de3b0d08-320f-4f4c-acb2-10fbe5ffbba8)

Reply:
Yep! We'll need to change the branch protection settings to match name of the new action for tests.